### ISSUE TITLE
Instructions on how to use Snappydata and ODBC

### ISSUE THREAD

**Issue by fire** [SnappyDataInc/snappydata#319]
How do you use odbc with Snappydata with the client on the Windows and the Snappydata server on Linux?

**Comment by sumwale** [comment]
@fire I will put up the notes and Windows installer in repository tomorrow.

**Comment by fire** [comment]
Thank you. I hope the Linux setup is also fairly simple.

Please add notes for that too.

I can imagine people using tableau and similar programs would be delighted. Personally I'll be testing it with http://erlang.org/doc/man/odbc.html.

**Comment by fire** [comment]
Have a great weekend. Hope to see it next week.

**Comment by piercelamb** [comment]
@sumwale what is the status of this?

**Comment by sumwale** [comment]
@fire @piercelamb Got stuck in various performance related issues. Still some loose ends for Windows installer. Will come back to this ASAP.

**Comment by fire** [comment]
Can't wait for this @sumwale. 

I'm having trouble finding ways to access SnappyData from Elixir or C++.

**Comment by sumwale** [comment]
@fire Are you trying from Linux/Mac or Windows? Have some pending work remaining for Windows installer but works fine as is on Linux (and should be fairly straightforward on Mac). Let me know the platform details (if Linux, then the flavour+version).

**Comment by fire** [comment]
I have two platforms:

One is centos 7 (the latest version). Hopefully you can describe the Linux setup.

```
$ cat /proc/version
Linux version 3.10.0-327.36.2.el7.x86_64 (builder@kbuilder.dev.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-4) (GCC) ) #1 SMP Mon Oct 10 23:08:37 UTC 2016
$ cat /etc/*elease
CentOS Linux release 7.2.1511 (Core) 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

CentOS Linux release 7.2.1511 (Core) 
CentOS Linux release 7.2.1511 (Core) 
```

The other is Windows 10.

If you can provide either or both, it'll unblock my work.

**Comment by fire** [comment]
I am also using a derivative of Ubuntu 16.04 called Elementary OS Loki 0.4 as my desktop, but I am currently running SnappyData under Centos 7 on a KVM virtual machine. So it's not a priority as I can also run my applications under Centos 7 vms.

**Comment by ashitabh** [comment]
Hi , please let us know if we can get ODBC driver at this point as I have to connect tableau to snappydata.

**Comment by piercelamb** [comment]
For others that find this thread, a SnappyData ODBC installer has been released as part of the [0.8 release](https://github.com/SnappyDataInc/snappydata/releases/tag/v0.8)

32bit: https://github.com/SnappyDataInc/snappydata/releases/download/v0.8/snappydata-0.8-odbc32.zip

64bit: https://github.com/SnappyDataInc/snappydata/releases/download/v0.8/snappydata-0.8-odbc64.zip

**Comment by fire** [comment]
Using odbc is straightforward, but comment!

**Comment by sumwale** [comment]
Doc link for ODBC driver usage: http://snappydatainc.github.io/snappydata/howto/#how-to-connect-using-odbc-driver

**Comment by sumwale** [comment]
@ashitabh @fire Please note that Tableau integration is still being actively worked on and does not work as is with the 0.8 release driver (due to some missing SQLGetInto APIs). It will be announced and separate release for ODBC driver made as soon as done.
